tests/torch/helpers.py

```diff
@@ -345,6 +345,7 @@ def __init__(self, input_size: Tuple, num_samples: int = 10):
         super().__init__()
         self._input_size = input_size
         self._len = num_samples
+        torch.manual_seed(42)
```

Comment:
Here is the seed = 42. In the ONNX seed is 0. Let's be more consistent. 
Also, multiple places with the same seed look not robust. IMHO, it would be better to collect such values in one variable.

Reply:
I agree it looks messy and not structured, but it shouldn't affect tests.
I put a common seed in tests/shared/definitions.py
And used it for all of my changes